crates/proof-of-sql/src/sql/ast/group_by_util.rs

```diff
@@ -114,8 +114,8 @@ pub(super) fn sum_aggregate_column_by_index_counts<'a, S: Scalar>(
         }
         Column::Scalar(col) => sum_aggregate_slice_by_index_counts(alloc, col, counts, indexes),
         Column::VarChar(_) => unimplemented!("Cannot sum varchar columns"),
-        Column::TimestampTZ(_, _, col) => {
-            sum_aggregate_slice_by_index_counts(alloc, col, counts, indexes)
+        Column::TimestampTZ(_, _, _) => {
+            unimplemented!("Cannot sum varchar columns")
```

Comment:
I wonder whether an error is better. This feature is inherently impossible to implement, not something we plan to implement in the future. Same for varchar.